pages/am/3.x/user-guide/user-management/user/user-guide-users.adoc

```diff
@@ -8,11 +8,117 @@
+You can also list users via the `/users` endpoint of the AM management API.
 
 image::am/current/graviteeio-am-userguide-um-search-users.png[]
 
-NOTE: Currently only users created by the AM management API or the SCIM protocol and external users who have been already logged can be found.
+NOTE: Currently only users created by the AM management API or the SCIM protocol and external users who have been already logged in can be found.
+
+==== List Users with a query (q)
+
+Finds users who match the specified query (Standard search mode).
+
+The value of `q` is matched against `username`, `email`, `displayName`, `firstName` or `lastName`.
+
+NOTE: In the Portal UI, you don't need to specify the `q` parameter but only the query value (i.e john doe).
+
+NOTE: The list operation performs a startsWith match query, you don't need to specify `firstName`, `lastName`, or `email` attribute name.
+
+===== Request Example
+
+----
+Request Example :
+
+curl -H "Authorization: Bearer :accessToken" \
+     -H "Content-Type:application/json;charset=UTF-8" \
+     -X GET \
+     http://GRAVITEEIO-AM-MGT-API-HOST/management/organizations/DEFAULT/environments/DEFAULT/:domain/gravitee/users?q=gravitee*&page=0&size=25
+----
+
+===== Response Example
+
+----
+{
+   "data":[
+      {
+         "id":"c5c22ed3-6a43-44c3-822e-d36a4324c3db",
+         "externalId":"5cf3527b-3aac-497a-b352-7b3aacf97a04",
+         "username":"gravitee.user@mail.com",
+         "email":"gravitee.user@mail.com",
+         "displayName":"Gravitee User",
+         "firstName":"Gravitee",
+         "lastName":"User",
+         "accountNonExpired":true,
+         "accountNonLocked":true,
+         "credentialsNonExpired":true,
+         "enabled":true,
+         "internal":true,
+         "preRegistration":false,
+         "registrationCompleted":true,
+         "referenceType":"domain",
+         "referenceId":"gravitee",
+         "source":"Default Identity Provider",
+         "loginsCount":0,
+         "webAuthnRegistrationCompleted":false,
+         "additionalInformation":{
+
+         },
+         "createdAt":1603037060752,
+         "updatedAt":1603037060752
+      }
+   ],
+   "currentPage":0,
+   "totalCount":1
+}
+----
+
+==== List Users with a filter (filter)
+
+Lists all users that match the filter criteria (Advanced search mode).
+
+When searching for users, you can create queries using link:https://tools.ietf.org/html/rfc7644#section-3.4.2.2[SCIM 2.0 query syntax] to refine your search.
+The search query must contain at least one valid expression with an attribute name followed by an attribute operator and an optional value.
+
+NOTE: Multiple expressions may be combined using the logical operators (`and` / `or`). Operators `[`, `]` and `not` are not supported.
+
+NOTE: In the Portal UI, you don't need to specify the `filter` parameter but only the query value (i.e displayName eq "john doe").
```

Comment:
NOTE: In AM Portal, you do not need to specify the `filter` parameter, only the query value (for example `displayName eq "john doe"`).